_source/_posts/2022-07-21-angular-security-xss.md

```diff
@@ -0,0 +1,402 @@
+---
+layout: blog_post
+title: "Protect Your Angular App from Cross-Site Scripting"
+author: alisa-duncan
+by: advocate
+communities: [security,javascript]
+description: "Dive into Cross-site Scripting (XSS) mitigation strategies and how to protect your Angular application from XSS attacks."
+tags: [security, angular]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness
+---
+
+In the last post of this SPA security series, we covered Cross-Site Request Forgery (CSRF) and how Angular helps you with a mitigation technique. 
+
+|Posts in the SPA web security series|
+| --- |
+| 1. [Defend Your SPA from Security Woes](/blog/2022/07/06/spa-web-security) |
+| 2. [Defend Your SPA from Common Web Attacks](/blog/2022/07/08/spa-web-securty-csrf-xss) |
+| 3. [Protect Your Angular App from Cross-Site Request Forgery](/blog/2022/07/19/angular-security-csrf) |
+| 4. **Protect Your Angular App from Cross-Site Scripting** |
+
+Next, we'll dive into Cross-Site Scripting (XSS) and look at the built-in security guards you get when using Angular.
+
+{% include toc.md %}
+
+## Cross-Site Scripting (XSS) protection
+
+In the [second post of this series](/blog/2022/07/08/spa-web-securty-csrf-xss), we covered an overview of Cross-Site Scripting (XSS). In summary, you learned that XSS occurs when code pollutes data and your application doesn't take safeguards to prevent the code from running.
```

Comment:
```suggestion
In the [second post of this series](/blog/2022/07/08/spa-web-securty-csrf-xss), we presented an overview of Cross-Site Scripting (XSS). In summary, you learned that XSS occurs when code pollutes data and your application doesn't take safeguards to prevent the code from running.
```
nitpick: because "cover an overview" seems a little dissonant